controllers/admin/AdminSaferPayOfficialPaymentController.php

```diff
@@ -261,7 +261,7 @@ protected function initForm()
         foreach ($paymentMethods as $paymentMethod) {
             $fields[] = [
                 'type' => 'free',
-                'label' => $this->l($paymentMethod),
+                'label' => $this->l($this->formatPaymentNotation($paymentMethod)),
```

Comment:
This translation would never work, nor it did previously. $this->l('Payment') is the only way to actually translate (with string inside, because of regex in PS)

Reply:
I don't get it why translations are needed for bank brand since MasterCard or UnionPay is translatable thing